src/main/java/org/web3j/protocol/admin/JsonRpc2_0Personal.java

```diff
@@ -0,0 +1,90 @@
+package org.web3j.protocol.admin;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.web3j.protocol.Web3jService;
+import org.web3j.protocol.admin.methods.response.NewAccountIdentifier;
+import org.web3j.protocol.admin.methods.response.PersonalListAccounts;
+import org.web3j.protocol.admin.methods.response.PersonalUnlockAccount;
+import org.web3j.protocol.core.JsonRpc2_0Web3j;
+import org.web3j.protocol.core.Request;
+import org.web3j.protocol.core.methods.request.Transaction;
+import org.web3j.protocol.core.methods.response.EthSendTransaction;
+
+/**
+ * JSON-RPC 2.0 factory implementation for common Parity and Geth.
+ */
+public class JsonRpc2_0Personal extends JsonRpc2_0Web3j implements Personal {
+
+    public JsonRpc2_0Personal(Web3jService web3jService) {
+        super(web3jService);
+    }
+    
+    @Override
+    public Request<?, PersonalListAccounts> personalListAccounts() {
+        return new Request<>(
+                "personal_listAccounts",
+                Collections.<String>emptyList(),
+                ID,
+                web3jService,
+                PersonalListAccounts.class);
+    }
+
+    @Override
+    public Request<?, NewAccountIdentifier> personalNewAccount(String password) {
+        return new Request<>(
+                "personal_newAccount",
+                Arrays.asList(password),
+                ID,
+                web3jService,
+                NewAccountIdentifier.class);
+    }   
+
+    @Override
+    public Request<?, PersonalUnlockAccount> personalUnlockAccount(
+            String accountId, String password,
+            BigInteger duration) {
+        List<Object> attributes = new ArrayList<>(3);
+        attributes.add(accountId);
+        attributes.add(password);
+        attributes.add(duration.longValue());
+        
+        return new Request<>(
+                "personal_unlockAccount",
+                attributes,
+                ID,
+                web3jService,
+                PersonalUnlockAccount.class);
+    }
+    
+    @Override
+    public Request<?, PersonalUnlockAccount> personalUnlockAccount(
```

Comment:
You shouldn't need to duplicate the logic in this second method - you should be able to use a single implementation, and override the implementation. Example here https://github.com/web3j/web3j/blob/master/src/main/java/org/web3j/protocol/parity/JsonRpc2_0Parity.java#L134